.github/README.md

```diff
@@ -117,9 +117,9 @@ You can find more examples in the [javalin-samples](https://github.com/javalin/j
 val app = Javalin.create { config ->
     config.http.defaultContentType = "application/json"
     config.http.generateEtags = true
+    config.http.brotliAndGzip()
```

Comment:
`config.http.brotliAndGzip()` should be `config.http.brotliAndGzipCompression()`

Reply:
I agree with @dzikoysk here, this is required.

Reply:
Oh, this is the readme.. we cannot update this until we release 6.0, so this should actually be removed :)